PartTwo/AVColorPickerSept12Sat/AVColorPickerCommunicationProtocol.h

```diff
@@ -0,0 +1,21 @@
+//
+//  AVColorPickerCommunicationProtocol.h
+//  AVColorPickerSept12Sat
+//
+//  Created by Ayuna Vogel on 9/12/15.
+//  Copyright © 2015 Ayuna Vogel. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol AVColorPickerCommunicationProtocol <NSObject>
+
+//- (void)socialMediaViewDidTapLikeButton:(SocialMediaView *)view;
+//
+//- (void)socialMediaViewDidTapCommentButton:(SocialMediaView *)view;
+//
+//- (void)socialMediaViewDidTapShareButton:(SocialMediaView *)view;
+
+-(void)didSelectColor:(UIColor *)color;
```

Comment:
Consider making this more general (the commented out methods above would work). `didSelectColor` implies intent, and you're `protocol` should not be aware of intent. The intent and logic is up to the object that conforms to the protocol.